lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -7811,10 +7811,29 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
     int InterpreterStackFrame::OP_GrowMemory(int32 delta)
     {
 #ifdef ENABLE_WASM
-        int32 oldPageCount = m_wasmMemory->GrowInternal((uint32)delta);
+
+        int32 oldPageCount = -1;
+        JavascriptExceptionObject* caughtExceptionObject = nullptr;
+        try
+        {
+            oldPageCount = m_wasmMemory->GrowInternal((uint32)delta);
```

Comment:
The try/catch should be done in GrowInternal around `TransferInternal` right now this only covers the grow_memory operator in the interpreter, it won't work in JIT and it won't work for the JS api on the WebAssembly.Memory